src/components/app/create/validationRules.ts

```diff
@@ -1,5 +1,20 @@
 import { PATTERNS } from '../../../config'
 
+const validateTagValue = (value: string): string[] => {
+    const errorList = []
+    if (value.length > 63) {
+        errorList.push('Can be max 63 characters')
+    }
+    let firstLastAlphanumeric = PATTERNS.START_END_ALPHANUMERIC.test(value)
```

Comment:
Make const. Also, make the changes for constant messaging as required

Reply:
done. The messaging changes will be done in next PR